src/main/features/first-contact/views/claim-summary.njk

```diff
@@ -37,6 +33,9 @@
           <p>{{ t('Find out about how we use and store your <a href="{{ link }}" target="_self">personal information</a>.', { link: AppPaths.privacyPolicyPage.uri }) | safe }}</p>
         </div>
       </details>
+
+      <p class="bold-small form-group-related">{{ t('Download the claim') }}</p>
+      <p><span class="block"><a href="{{ DashboardPaths.claimReceiptReceiver.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Download claim') }}</a>{{ t(' (PDF)') }}</span></p>
```

Comment:
Dashboard paths should not be used in first contact feature. Use `ClaimPaths.receiptReceiver` directly.

Reply:
Remove space from `' (PDF)`. Make that space in HTML so that we do not have to have whitespace in `*.po` files.